src/fuzz_introspector/html_helpers.py

```diff
@@ -52,8 +53,24 @@ def html_table_add_row(elems: List[Any]) -> str:
 
 def html_get_header(calltree: bool = False,
                     title: str = "Fuzz introspector") -> str:
-    header = """<html>
+    gtag_tracking = ""
+    try:
+        gtag = os.environ['G_ANALYTICS_TAG']
+        gtag_tracking += f"""<!-- Google tag (gtag.js) -->
+                <script async src="https://www.googletagmanager.com/gtag/js?id=G-{gtag}"></script>
```

Comment:
Should we prefix the tag with `G-` or let the user set this themselves?

Reply:
The prefix might not be G- AFAIK so we should not prefix it. Updated.